modules/core/src/controllers/map-controller.js

```diff
@@ -306,6 +306,21 @@ class MapState extends ViewState {
     });
   }
 
+  shortestPathFrom(viewState) {
```

Comment:
The controller base class could add an empty implementation in the `// PROTECTED` section so that someone reading that class can easily see what options they have to configure this class.

Reply:
It is part of the ViewState base class.